src/server/db/db_helper.py

```diff
@@ -438,9 +438,15 @@ async def validate_sql(self, db_name, query):
             # return False
 
 class sqlite_database(Database):
+    def __init__(self):
+        self.temp_db_folder = "temporary_db"
+        if not os.path.exists(self.temp_db_folder):
+            os.makedirs(self.temp_db_folder)
+
     async def get_connection(self):
         try:
-            con = sqlite3.connect("temp.db")
+            db_file_path = os.path.join(self.temp_db_folder, "temp.db")
```

Comment:
I believe you are not using this function anywhere so dont use temp.db to create a connection. Dont define this function if you are not using it

Reply:
I have removed it and added a pass statement, because the function itself may be called at times, but it does not do anything, completely removing it may cause issues in app.py itself.